maestro-cli/src/main/java/maestro/cli/idb/IdbCompanion.kt

```diff
@@ -0,0 +1,137 @@
+package maestro.cli.idb
+
+import com.github.michaelbull.result.Ok
+import com.github.michaelbull.result.Result
+import com.github.michaelbull.result.runCatching
+import idb.CompanionServiceGrpc
+import idb.HIDEventKt
+import idb.Idb
+import idb.hIDEvent
+import idb.point
+import io.grpc.ManagedChannel
+import io.grpc.ManagedChannelBuilder
+import ios.grpc.BlockingStreamObserver
+import maestro.cli.device.Device
+import maestro.debuglog.DebugLogStore
+import maestro.utils.MaestroTimer
+import java.net.Socket
+import java.util.concurrent.TimeUnit
+import kotlin.concurrent.thread
+
+object IdbCompanion {
+    private val logger = DebugLogStore.loggerFor(IdbCompanion::class.java)
+
+    // TODO: Understand why this is a separate method from strartIdbCompanion
```

Comment:
@amanjeetsingh150 Do you know the details on why this was implemented differently than `startIdbCompanion`?